### DISCUSSION TITLE
WinUI 3 C++ - Remove IDL file under App.xaml

### DISCUSSION THREAD

**Discussion by jaigak** [microsoft/WindowsAppSDK#1886] [General]
In the current project templates for C++ WinUI 3, there's an empty IDL file under the App.xaml node in Solution Explorer. However, since it's empty, I would suggest removing it.

**Comment by shelllet** [comment]
please don't do that,  i like put all other idl contents to app.idl.

many idl file would take more compile time.

**Reply by jaigak** [reply]
You can always create one later but it shouldn't be by default if it's empty.
